napari_svg/layer_to_xml.py

```diff
@@ -153,7 +159,9 @@ def points_to_xml(data, meta):
     """
     # Extract metadata parameters
     if 'size' in meta:
-        size = np.mean(meta['size'], axis=1)
+        size = meta['size']
+        if size.ndim == 2:
+            size = np.mean(size, axis=1)
```

Comment:
What's this for? Probably warrants a comment

Reply:
That's from #27 for backward compat.